scripts.py

```diff
@@ -999,16 +1055,29 @@ def arg_parser(cls):
             '--license', help='The url location of the licensing document for this feed'
         )
         parser.add_argument(
+            '--storage-bucket', help='The S3 bucket to which static feeds should be uploaded'
+        )
+        parser.add_argument(
             '--search-url', help='Upload to this elasticsearch url. elasticsearch-index must also be included'
         )
         parser.add_argument(
             '--search-index', help='Upload to this elasticsearch index. elasticsearch-url must also be included'
         )
         return parser
 
+    @property
+    def library(self):
+        """Provides a predictable library to use when creating static feeds"""
+        library = self.__library
+        if not library:
+            library = Library.instance(self._db)
```

Comment:
The method still exists in multi-tenant core right now, but I'll stop using it here if the goal is to remove it eventually. Naming it after the app makes sense.